src/pages/public/about-us-page/support-section/components/support-card/SupportCard.tsx

```diff
@@ -0,0 +1,15 @@
+interface SupportCardProps {
+    IMG: string;
+    ALT: string;
+    DESCRIPTION: string;
+    index: number;
+}
+
+export function SupportCard({ IMG, ALT, DESCRIPTION, index }: SupportCardProps) {
+    return (
+        <div key={index} className={`support-card card-${index + 1}`}>
```

Comment:
A component should never apply a `key` to itself internally.
```suggestion
        <div className={`support-card card-${index + 1}`}>
```